packages/table/src/tableBody.tsx

```diff
@@ -133,7 +133,6 @@ export class TableBody extends React.Component<ITableBodyProps, {}> {
 
     private activationCell: ICellCoordinates;
     private batcher = new Batcher<JSX.Element>();
-    private isRenderingBatchedCells = false;
```

Comment:
This variable used to seem relevant, but it wasn't (and it was never being set to `true` anyway, whoops). When `renderMode === RenderMode.BATCH`, you can guarantee the batched rendering will trigger; so all you care about from there is whether it's done yet.